engine/sql/082_platform.sql

```diff
@@ -0,0 +1,29 @@
+-- +migrate Up
+CREATE TABLE platform_model (
+  id BIGSERIAL PRIMARY KEY,
+  name VARCHAR(50),
+  author VARCHAR(50),
+  identifier VARCHAR(150),
+  icon VARCHAR(20),
+  default_config JSONB,
+  disabled BOOLEAN,
+  hook BOOLEAN,
+  file_storage BOOLEAN,
+  block_storage BOOLEAN,
+  deployment BOOLEAN,
+  compute BOOLEAN
+);
+
+CREATE TABLE project_platform (
+  id BIGSERIAL PRIMARY KEY,
```

Comment:
why this association table have an id ? shouldn't the primary key composed by model_id and project_id ?

Reply:
you can add the same model any time you want,